torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19259,6 +19278,15 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     PythonRefInfo(
         "_refs.addcmul",
         torch_opinfo_name="addcmul",
+        skips=(
+            # Reference result was farther (1.3343989849090576e-05)
+            # from the precise computation than the torch result
+            # was (9.592622518539429e-06)!
+            DecorateInfo(unittest.skip("Unstable expected failure. Skipped!"), 'TestCommon', 'test_python_ref',
```

Comment:
yes, it is flaky: some CI jobs fail while some others don't.

Reply:
Correct me if I'm wrong here, but the purpose of this test is to check that the Python reference implementation is doing the same thing as the C++ implementation, right? If so, isn't the exact result irrelevant here? It seems that the correctness of the C++ op is checked elsewhere.

If that is the case, I don't think we should skip here, but rather increase the tolerances to avoid the flakiness. Otherwise the two implementations can drift apart and we would never know. Granted, with tolerances set they can also drift a little, but that is a lot more safety than a blanket skip.

Reply:
Yes, you are right. However, changing the tolerances is not implemented in the python_ref tests:
https://github.com/pytorch/pytorch/blob/752e43c301ac433b909c21e2b34bc3428afbb173/test/test_ops.py#L441
and it would be OT for this PR.

Reply:
That is unfortunate. In that case could you change the comment to a `FIXME` explaining that we should set tolerances there, but the test doesn't support it?

Reply:
Added FIXME comment.